ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/ovn/sdn2ovn/OWNERS

```diff
@@ -0,0 +1,14 @@
+approvers:
+- jianlinliu
+- yunjiang29
```

Comment:
Please replace `yunjiang29` with `jianli-wei` since Jianli is the owner of GCP installation. And please help update other related files as well. Thanks.